diff --git a/drivers/Kconfig b/drivers/Kconfig
index efb66e25fa2d..9588ab648afc 100644
--- a/drivers/Kconfig
+++ b/drivers/Kconfig
@@ -1,6 +1,8 @@
 # SPDX-License-Identifier: GPL-2.0
 menu "Device Drivers"
 
+source "drivers/simple_trace/Kconfig"
+
 # Keep I/O buses first
 
 source "drivers/amba/Kconfig"
diff --git a/drivers/Makefile b/drivers/Makefile
index 1bec7819a837..74363d78710e 100644
--- a/drivers/Makefile
+++ b/drivers/Makefile
@@ -5,6 +5,7 @@
 # 15 Sep 2000, Christoph Hellwig <hch@infradead.org>
 # Rewritten to use lists instead of if-statements.
 #
+obj-y				+= simple_trace/
 
 # Some driver Makefiles miss $(srctree)/ for include directive.
 ifdef building_out_of_srctree
